regression-v1x10.sh

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
```

Comment:
@macpijan aren't we going into direction of a single script?

Reply:
Correct, https://github.com/Dasharo/open-source-firmware-validation/pull/208#discussion_r1511260446